profile/arc_cwl_ro_crate.md

```diff
@@ -23,11 +23,14 @@ To continue staying consisten with the [ISA RO-Crate Profile](https://github.com
 
 ```mermaid
 flowchart TD
-        A["File\nSoftwareSourceCode\nComputationalWorkflow"] -- "input\noutput" --> B["FormalParameter"]
-        A -- "instrument" --> C["CreateAction"]
+        A["File\nSoftwareSourceCode\nComputationalWorkflow\nLabProtocol"] -- "input\noutput" --> B["FormalParameter"]
+        A -- "instrument" --> C["CreateAction\nLabProcess"]
+        C -- "executes" --> A
         C -- "agent" --> D["Person or Organization"]
         B -- "exampleOfWork" --> E["File or Property Value"]
         C -- "object result" --> E
```

Comment:
Why are `File` and `PropertyValue` merged here? To me, it doesn't make sense for the `object`/`result` to be a `PropertyValue`.

Reply:
This is part of the original process run crate:
Entities referenced by an action’s [object](http://schema.org/object) or [result](http://schema.org/result) SHOULD be of type File (an RO-Crate alias for [MediaObject](http://schema.org/MediaObject)) for files, [Dataset](http://schema.org/Dataset) for directories and [Collection](http://schema.org/Collection) for [multi-file datasets](https://www.researchobject.org/workflow-run-crate/profiles/0.5/process_run_crate/#representing-multi-file-objects), but MAY be a [CreativeWork](http://schema.org/CreativeWork) for other types of data (e.g. an online database); they MAY be of type [PropertyValue](http://schema.org/PropertyValue) to capture numbers/strings that are not stored as files.

Reply:
That's interesting, and also makes a lot of sense for some kinds of workflows. It doesn't fit with the ISA model though. However, in an ARC, CWL workflows are clearly separated from the ISA process graph. So I'm not sure how we should handle this. Do we want to allow/encourage such workflows that do not produce files? If we do not, it is ok to diverge in a derived profile.

Any opinions on that @HLWeil, @muehlhaus?

Reply:
I think it makes sense in processing steps/workflows, that are part of a wf/nested wf. But in the ARC, we recommend to annotate higher level wfs (see [here](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile/blob/0.1-draft/profile/arc_cwl_ro_crate.md#workflow-run-crate-in-arcs). Those shouldn't return results that are not files.